ferretdb/ferretdb.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/FerretDB/FerretDB/internal/clientconn"
+	"github.com/FerretDB/FerretDB/internal/handlers/registry"
+	"github.com/FerretDB/FerretDB/internal/util/logging"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/version"
+)
+
+// Config contains a PostgreSQL backend connection string.
```

Comment:
We should either change the comment to something like this:
```suggestion
// Config contains backend connection strings.
```
Or remove `TigrisURL`.

Reply:
We can't remove TigrisURL :)

Reply:
I mean we don't use it anywhere here. So we can just pass an empty string to the `NewHandlerOpts` initialization.

Reply:
> I mean we don't use it anywhere here. So we can just pass an empty string to the `NewHandlerOpts` initialization.

TigrisURL is used to create a Tigris handler.

Comment changed.